spec/requests/topics/index_spec.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'GET /topics', type: :request do
+  let(:user) { create :user }
+
+  subject do
+    get '/topics', headers: headers
```

Comment:
remember you can use the helper instead of '/topics'

Reply:
I completely forgot, I'll update.